iphone/Classes/TiUIWebViewConfigurationProxy.h

```diff
@@ -0,0 +1,34 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2017 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+#ifdef USE_TI_UIWEBVIEW
+
+#import "TiProxy.h"
+
+@interface TiUIWebViewConfigurationProxy : TiProxy {
```

Comment:
Noting one more time: Move to iOS-only `TiUIiOSWebViewConfigurationProxy` namespace.